404.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Climate Hub 404 Page
+ *
+ * @package climatehub
+ */
+?>
+
+<?php get_header(); ?>
+  
+  <div class="container my-5">
+    <div class="row">
+      <div class="col-12 text-center">
+        <div class="h1 font-weight-bold">404</div>
+        <div class="h2">Page Not Found</div>
+        <div class="h3">
+          We can't find the page you're looking for. Go back or head over the the homepage.
```

Comment:
Can you make this into a custom field or something, I think it would be better if they could decide what to have here so that it aligns with their brand.

Reply:
Sure, I'll make the change